From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: cyrozap <cyrozap@gmail.com>
Date: Tue, 11 Jun 2024 20:50:30 -0500
Subject: [PATCH] 6633: RISCV: Fix trivial typo in RISCV:LE:64:default language
 description

RISCV: Fix typo in RISCV:LE:64:default language definition
---
 Ghidra/Processors/RISCV/data/languages/riscv.ldefs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Ghidra/Processors/RISCV/data/languages/riscv.ldefs b/Ghidra/Processors/RISCV/data/languages/riscv.ldefs
index f7d946ec63..737f9ed120 100644
--- a/Ghidra/Processors/RISCV/data/languages/riscv.ldefs
+++ b/Ghidra/Processors/RISCV/data/languages/riscv.ldefs
@@ -83,7 +83,7 @@
             slafile="riscv.lp64d.sla"
             processorspec="RV64GC.pspec"
             id="RISCV:LE:64:default">
-    <description>RISC-V 32 little default</description>
+    <description>RISC-V 64 little default</description>
     <compiler name="gcc" spec="riscv64-fp.cspec" id="gcc"/>
     <external_name tool="DWARF.register.mapping.file" name="riscv64.dwarf"/>
     <external_name tool="gnu" name="riscv:rv64"/>
-- 
2.45.1

